### PR TITLE
Feature limited transfer

### PR DESCRIPTION
**Pull request by davy-vr**

### CODE REVIEW

File: contracts/token/LimitedTransferMiniMeToken.sol [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+  address public contractAddress;
+
+  mapping (address => bool) public limitedTransferAddresses;
+
+  function LimitedTransferMiniMeToken(address _tokenFactory, address _parentToken, uint _parentSnapShotBlock, string _tokenName, uint8 _decimalUnits, string _tokenSymbol, bool _transfersEnabled) MiniMeToken(_tokenFactory, _parentToken, _parentSnapShotBlock, _tokenName, _decimalUnits, _tokenSymbol, _transfersEnabled) {
+    //constructor
+  }
+
+  function addLimitedTransferAddress(address _allowed) public onlyController returns (bool success) {
```
Comment (Qkyrie) [inline contracts/token/LimitedTransferMiniMeToken.sol]: can we change this to (address, bool), so we can disallow someone afterwards?

File: contracts/token/LimitedTransferMiniMeToken.sol [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  }
+
+  function enableLimitedTransfers(bool _limitedTransfersEnabled) public onlyController {
+    limitedTransfersEnabled = _limitedTransfersEnabled;
+    if (_limitedTransfersEnabled) {
+      super.enableTransfers(_limitedTransfersEnabled);
+    }
+  }
+
+  function approve(address _spender, uint256 _amount) public canTransfer(msg.sender, _spender) returns (bool success) {
```
Comment (Qkyrie) [inline contracts/token/LimitedTransferMiniMeToken.sol:41]: do we need to include the "canTransfer" as modifier? 
What if we always allow approve? 
Is there a use case where we need to transfer from an arbitrary account to the contract where the initiator (msg.sender) is not the contract?

The current flow only uses approve (from any account) and transferFrom/transfer (from contract)

File: contracts/token/FundRequestToken.sol [hunk 3]
```diff
@@ -1,24 +1,23 @@
...
+import "./LimitedTransferMiniMeToken.sol";
 
 
 /**
  * Fundrequest Token as deployed by https://fundrequest.io
  * Davy Van Roy
  * Quinten De Swaef
  */
-contract FundRequestToken is MiniMeToken {
+contract FundRequestToken is LimitedTransferMiniMeToken {
```
Comment (Qkyrie) [inline contracts/token/FundRequestToken.sol:12]: Can we somehow change this to a Minime Token with a TransferCapability? So we can have composition over inheritance? Not necessary, but could be nice

**Review by Qkyrie — COMMENTED** [review]
Loving the tests!

File: contracts/token/LimitedTransferMiniMeToken.sol (hunk 2 above)
Comment (davy-vr) [reply contracts/token/LimitedTransferMiniMeToken.sol:41]: As long the transferFrom still has this modifier it should be fine indeed

**Pull request merged**